src/components/Checkbox/Checkbox.tsx

```diff
@@ -1,48 +1,44 @@
-import {
-  ChangeEvent,
-  ComponentPropsWithoutRef,
-  FC,
-  ReactElement
-} from 'react';
+import { ChangeEvent, ComponentPropsWithoutRef, DragEvent, FC, ReactElement, useState } from 'react';
 import { ThemeProvider } from '@emotion/react';
 import { StyledCheckbox, StyledLabel } from './checkboxStyles';
 import { ReactComponent as Check } from './assets/check.svg';
 import { ReactComponent as Indeterminate } from './assets/indeterminate.svg';
 import theme from 'src/styles/theme';
 
 export interface CheckboxProps {
-  /**
-   * The value to control whether the checkbox should be checked or not.
-   */
-  checked?: boolean;
-  /**
-   * The svg icon to display when checked is true. If left undefined, a check mark will be displayed.
-   */
-  checkedIcon?: ReactElement<SVGElement>;
-  /**
-   * A display state in which it is unknown whether checked should be true or false.
-   */
-  indeterminate?: boolean;
-  /**
-   * The (optional) label for the checkbox.
-   */
-  label?: string | ReactElement;
-  /**
-   * HTML input element disabled prop.
-   */
-  disabled?: boolean;
-  /**
-   * The action that should be run when the checked state changes.
-   */
-  onChange?: (event: ChangeEvent<HTMLInputElement>) => void;
-  /**
-   * The classes to pass to the checkbox.
-   */
-  className?: string;
-  /**
-   * The classes to pass to the checkbox label.
-   */
-  labelClassName?: string;
+    /**
```

Comment:
why the reformat with 4 spaces?

Reply:
the prettier.rc file didn't have the right settings, i adjusted it.